pkg/debug/scripts/collect-info.sh

```diff
@@ -216,6 +216,37 @@ collect_pillar_backtraces()
 
     echo "- done pillar backtraces"
 }
+collect_zfs_info()
+{
+        type=$(cat /run/eve.persist_type)
+        if [ "$type" = "zfs" ]; then
+           echo "- Collecting ZFS specific info"
```

Comment:
Is the indentation width different or is it just github showing it weirdly?

Reply:
Yeah that's odd.

Reply:
I've checked out the PR, and yes - it differs. 8 spaces instead of 4 are used.